android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -12,6 +12,8 @@
 import android.os.Bundle;
 
 public class TagDispatcher {
+    private static final int DELAY_PRESENCE = 5000;
```

Comment:
@petter-fidesmo That would require a very extensive phone model database and name matching rules - not worth the effort at this point in time :)